File: fixtures/scale/swift/medium/Functions/TrigFunctions.swift
import Foundation

enum TrigFunctions {
    static func install(_ registry: FunctionRegistry) {
        registry.define("sin") { sin($0[0]) }
        registry.define("cos") { cos($0[0]) }
        registry.define("tan") { tan($0[0]) }
        registry.define("asin") { asin($0[0]) }
        registry.define("acos") { acos($0[0]) }
        registry.define("atan") { atan($0[0]) }
    }
}

File: fixtures/scale/java/medium/src/calc/funcs/TrigFunctions.java
package calc.funcs;

import calc.config.Settings;
import calc.eval.FunctionRegistry;

public class TrigFunctions {
    public static void register(FunctionRegistry registry, Settings settings) {
        registry.addUnary("sin", x -> Math.sin(settings.toRadians(x)));
        registry.addUnary("cos", x -> Math.cos(settings.toRadians(x)));
        registry.addUnary("tan", x -> Math.tan(settings.toRadians(x)));
    }
}
